recipes/lfortran/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+printenv
+
+if [[ ${MACOSX_DEPLOYMENT_TARGET} != "" ]]; then
+    # macOS
+    python -m pip install . -vv
+else
+    # Linux
+    python -m pip install . -vv
```

Comment:
what is different in these bodies? and shouldn't it be $PYTHON?

Reply:
It used to be different as we experimented with various approaches. Now it is the same, so I am going to fold this into the meta.yaml file itself.

Reply:
Fixed in 223322eec8122540295d647230f1ff48d21b88bb.